client/js/widgets/button.js

```diff
@@ -141,7 +141,11 @@ class Button extends Widget {
         const count = a.count || 999999;
 
         if(isValidID(a.from) && isValidID(a.to)) {
-          if(a.face === null && typeof a.from == 'string' && typeof a.to == 'string' && !widgets.get(a.to).children().length && widgets.get(a.from).children().length <= count) {
+          const item = widgets.get(a.from)
+          const dest = widgets.get(a.to)
+          if (item && item.p('type') == null && dest.p('type') == 'holder') {
```

Comment:
At least for `dest` this should error if `a.to` is an array.

Reply:
I also think this shouldn't be `== null` but rather `!= 'holder'`. Any reason behind this?

Reply:
fixed